lib/features/judge/presentation/pages/create_judge_screen.dart

```diff
@@ -0,0 +1,181 @@
+import 'package:fiura_ecosystem/core/entities/judge_entity/judge_entity.dart';
+import 'package:fiura_ecosystem/features/judge/presentation/cubit/judge_state.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../../../dependencies.dart';
+import '../../../utils/validators.dart';
+import '../cubit/judge_cubit.dart';
+
+class CreateJudgeScreen extends StatefulWidget {
+  const CreateJudgeScreen({super.key});
+
+  @override
+  State<CreateJudgeScreen> createState() => _CreateJudgeScreenState();
+}
+
+class _CreateJudgeScreenState extends State<CreateJudgeScreen> {
+  // Form key
+  final _formKey = GlobalKey<FormState>();
+  // TextField controllers
+  final controllerJudgeName = TextEditingController();
+  final controllerJudgeAbout = TextEditingController();
+  final controllerJudgeFacebook = TextEditingController();
+  final controllerJudgeTwitter = TextEditingController();
+  final controllerJudgeInstagram = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text('Registro de nuevo Juez'),
+        ),
+        body: BlocProvider(
+          create: (_) => getIt<JudgeCubit>(),
+          child: BlocConsumer<JudgeCubit, JudgeState>(
+              listener: (context, snapshot) {
+            if (snapshot is Loading) {
+              ScaffoldMessenger.of(context).showSnackBar(
+                const SnackBar(
```

Comment:
Because we are using freeze you can simplify this nig if statement with snapshot. MeabyMap

Reply:
Fixed